src/vivarium_gates_nutrition_optimization_child/constants/data_keys.py

```diff
@@ -142,12 +138,19 @@ class __ProteinEnergyMalnutrition(NamedTuple):
         "cause.protein_energy_malnutrition.restrictions"
     )
 
+    @property
```

Comment:
In MNCH Child, PEM is in the data keys, but not built in the artifact. MNCH loads data for both severe and moderate PEM separately, but the CIFF SAM wasting model just uses the more general PEM. It seemed best to just include PEM in the artifact as well.